src/components/tooltip/__stories__/index.story.jsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Tooltip, Button } from '@/index';
+
+export const Basic = () => {
+  const options = {
+    tooltip: 'An awesome tooltip',
+    hoverable: true,
+    position: 'bottom',
+  };
+
+  return (
+    <Tooltip {...options}>
+      <Button>bottom</Button>
+    </Tooltip>
+  );
+};
+
+export const Position = () => {
+  const positions = [
+    'top',
+    'top-start',
+    'top-end',
+    'bottom',
+    'bottom-start',
+    'bottom-end',
+    'left',
+    'right',
+    'left-start',
+    'right-start',
+    'left-end',
+    'right-end',
+  ];
+
+  return (
+    <div className="pb-6 Row">
+      {positions.map((pos, ind) => {
+        return (
+          <div key={ind} className="mr-13 mt-8">
+            <Tooltip placement={pos} tooltip="An awesome tooltip" hoverable={true}>
+              <Button>{pos}</Button>
+            </Tooltip>
+          </div>
+        );
+      })}
+    </div>
+  );
+};
+
```

Comment:
Please create one story to show both the "hoverable" & "click" behavior of tooltip